posts/01-first-map/earthlab-test-student/uttc-map.md

```diff
@@ -0,0 +1,5 @@
+# United Tribes Technical College preserves the culture of Native Peoples of the Upper Midwest through education
```

Comment:
I like that this title is in a headline style, and lends some meaning to the map. It's not totally clear what kind of education you're talking about and how that helps preserve culture. Looks good overall!